src/serviceFacades/bags.js

```diff
@@ -0,0 +1,222 @@
+import { useMutation, useQuery, useQueryCache } from "react-query";
+
+import callAPI from "../common/callApi";
+
+export const DEFAULT_BAG_QUERY_KEY = "getDefaultBag";
+export const DEFAULT_BAG_MUTATION_KEY = "mutateDefaultBag";
+export const DEFAULT_BAG_DELETE_KEY = "deleteDefaultBag";
```

Comment:
I may have missed it, and this is minor, but I don't think these keys are used outside of this file so they may not need to be exported.  Also, I'm not sure the mutation or delete key are used.  IIRC, the mutation queries don't require a key.

Reply:
Thanks for pointing this out. Originally I was just using bare useQuery() and useMutation() calls and then changed to the current setup. Forgot to unexport stuff and remove uneeded keys.